plugins/destination/postgresql/client/migrate.go

```diff
@@ -342,6 +342,9 @@ func (c *Client) createTableIfNotExist(ctx context.Context, table *schema.Table)
 		}
 		columnName := pgx.Identifier{col.Name}.Sanitize()
 		fieldDef := columnName + " " + pgType
+		if col.Name == "_cq_id" {
```

Comment:
maybe we can use cq id col from schema pkg?
```suggestion
		if col.Name == schema.CqIdColumn.Name {
```

Reply:
Ah, shoot. I'll fix that in a bit